table/get_context.h

```diff
@@ -74,6 +74,7 @@ class GetContext {
     kCorrupt,
     kMerge,  // saver contains the current merge result (the operands)
     kUnexpectedBlobIndex,
+    kUnexpectedWideColumnEntity,
```

Comment:
Will this be removed in the future if wide column is generally supported in RocksDB? If so, maybe put a TODO.